cost_param: {r_safe: 0.5,ground_reject_height: 0.5}

optim_param: {descending_rate: 0.4,
                descending_rate_min: 0.3,
                descending_rate_max: 0.7,
                n_step: 11,
                max_iter: 100,
                weight_prior: 0.5, 
                term_cond: 0.0005,
                gamma: 10.0
                }

pred_param: {prediction_horizon: 4,
             observation_size: 4,
             observation_horizon: 4,trigger_tol_accum_error : 0.5}
             
target_waypoint: {x: [3.4578, 6.7478, 8.3223, 14.0,  11.4981], y:[4.1620, 8.0200, 3.9896,2 ,8.0515]}

